vkcs/db_instance_client.go

```diff
@@ -570,6 +621,26 @@ func clusterGetCapabilities(client databaseClient, id string) (r getInstanceCapa
 	return
 }
 
+func instanceGetBackupSchedule(client databaseClient, id string) (r getInstanceBackupScheduleResult) {
+	reqOpts := getDBRequestOpts(200)
+	var result *http.Response
+	result, r.Err = client.Get(backupScheduleURL(client, instancesAPIPath, id), &r.Body, reqOpts)
+	if r.Err == nil {
+		r.Header = result.Header
+	}
+	return
+}
+
+func clusterGetBackupSchedule(client databaseClient, id string) (r getClusterBackupScheduleResult) {
```

Comment:
should not we need to move this to cluster client?

Reply:
Moved